aopy/utils.py

```diff
@@ -429,4 +431,54 @@ def calc_radial_dist(pos, origin=(0,0)):
     for i, p in enumerate(pos):
         dist[i] = math.dist(origin, p)
     
-    return dist
\ No newline at end of file
+    return dist
+
+'''
+Memory
+'''
+def get_memory_available_gb():
+    '''
+    Get the available system memory in gigabytes. Only works on linux platforms.
+
```

Comment:
Just added some more info: The results of this function are equivalent to this command in the terminal. `grep MemAvailable /proc/meminfo ` -> available memory 
`grep MemTotal /proc/meminfo`  -> total memory